src/Humanizer.Tests/Localisation/hu/TimeSpanHumanizeTests.cs

```diff
@@ -5,7 +5,7 @@ public class TimeSpanHumanizeTests
 {
     [Theory]
     [Trait("Translation", "Google")]
-    [InlineData(366, "egy év")]
```

Comment:
can you explain this change. `egy év` seems correct

Reply:
Hello,

This is because I have addedd the missing hungarian translations to the resource file. `egy év` is correct, when `toWords` is set to `true`, othervise the number will be number, not words (`1` vs `egy`).

I created new test cases to verify the word representation of TimeSpans.

https://github.com/Humanizr/Humanizer/pull/1501/files#diff-6829957358e12fc8868dd8833601a3630bc4d27a0489053daa4d863c2459dc7dR612